### ISSUE TITLE
Chat translate not working

### ISSUE THREAD

**Issue by joshuarodriguez82** [amazon-connect/ai-powered-speech-analytics-for-amazon-connect#34]
I uploaded the template, followed the instructions on the blog post and admin guide but cannot get chat or audio transcripts to work. Currently the chat comes in, there is no detection in the custom CCP at all. I have a test chat to the aiChatFlow that comes in the s3 bucket. Please let me know what other information you need to help diagnose the issue.

**Comment by mcosicaws** [comment]
Could you please check your browser's console for any errors?

**Comment by kqia040** [comment]
Hey @joshuarodriguez82 have you had the chance to check?

**Comment by joshuarodriguez82** [comment]
Yes sorry for the delayed response, I'm getting quite a few browser errors that I do not know how to diagnose. I have them saved as text files. 



[d3v8smcgg7of5m.cloudfront.net-1639089973052-SACCPErrors.log](https://github.com/amazon-connect/ai-powered-speech-analytics-for-amazon-connect/files/7688928/d3v8smcgg7of5m.cloudfront.net-1639089973052-SACCPErrors.log)
[d3v8smcgg7of5m.cloudfront.net-1639089941098-SACCPErrors.log](https://github.com/amazon-connect/ai-powered-speech-analytics-for-amazon-connect/files/7688929/d3v8smcgg7of5m.cloudfront.net-1639089941098-SACCPErrors.log)
[d3v8smcgg7of5m.cloudfront.net-1639089985379-SACCPErrors.log](https://github.com/amazon-connect/ai-powered-speech-analytics-for-amazon-connect/files/7688930/d3v8smcgg7of5m.cloudfront.net-1639089985379-SACCPErrors.log)
